accounts/urls.py

```diff
@@ -0,0 +1,22 @@
+from django.urls import path, reverse_lazy
+from django.contrib.auth import views as auth_views
+
+from . import views
+
+
+app_name = 'accounts'
```

Comment:
Lets remove it.

Reply:
tried removing these but the code broke as im using urls that are namespaced within an app. helps avoid conficts in case of two apps/sub apps having same url name can address them as app_name:url_name etc

Reply:
include((pattern_list, app_namespace), namespace=None)